### ISSUE TITLE
Template front

### ISSUE THREAD

**Issue by andressegeo** [K-MASH/PlayMusic#1]
Benchmark sur les solutions existantes
Retenir la solution qui répond le plus à nos besoins( don't forget le lecteur)

**Comment by tchafack** [comment]
J'ai check ça ce matin et j'ai trouvé quelques wé.
* Il y a des fichiers PSD pour une intégration dans le projet Android
* J'ai trouvé un template (Material UI Template 3.0) mais il est payant et c'est le best et le plus easy que j'ai trouvé pour mettre en place. https://codecanyon.net/item/android-material-ui-template-21/13971332?_ga=2.224813784.378294150.1524491159-452524208.1518529883

**Comment by andressegeo** [comment]
C'est payant, non?
N'y à t'il vraiment rien eu de free?

**Comment by andressegeo** [comment]
https://code.tutsplus.com/tutorials/build-a-music-app-with-an-android-app-template--cms-29966

**Comment by tchafack** [comment]
oui c'est payant. Le lien que tu as send est juste un p'tit tuto sur comment use ça

**Comment by andressegeo** [comment]
J'ai trouvé ceci: 
**Free**:
https://www.sketchappsources.com/free-source/1676-android-music-player-app-sketch-freebie-resource.html

https://www.sketchappsources.com/free-source/994-material-design-music-player-sketch-freebie-resource.html


**Payant**
https://www.template.net/design-templates/music-app-designs/

**Comment by tchafack** [comment]
Pour les trucs free, je look encore. Pour le moment, il y a juste que des trucs PSD (photoshop). Mais j'ai encore look leur intégration

**Comment by tchafack** [comment]
Oui, j'ai déjà vu ce que tu as send. Ce sont des projets au format sketch. Ce sont uniquement des maquettes, comme un truc PSD

**Comment by tchafack** [comment]
Je check le dernier lien

**Comment by tchafack** [comment]
J'ai trouvé une autre solution https://github.com/googlesamples/android-UniversalMusicPlayer
elle n'a pas de cool design mais je crois que les features techniques sont très importantes.
Je pense qu'on peut intégrer/jumeler ce projet et qq templates PSD que j'ai trouvé (https://creativemarket.com/SherbakYurii/578345-Garnet?u=templatenet | https://creativemarket.com/cmakyrk/460454-Soundify-Mobile-Music-App-Design?u=templatenet)
